subsys/bluetooth/controller/hci/hci.c

```diff
@@ -5759,51 +5759,19 @@ int hci_iso_handle(struct net_buf *buf, struct net_buf **evt)
 			return -EINVAL;
 		}
 
-		/* Get free node tx */
-		tx = ll_iso_tx_mem_acquire();
-		if (!tx) {
-			BT_ERR("ISO Tx Buffer Overflow");
-			data_buf_overflow(evt, BT_OVERFLOW_LINK_ISO);
-			return -ENOBUFS;
-		}
-
-		struct pdu_bis *pdu = (void *)tx->pdu;
-
-		/* FIXME: Update to use correct LLID for BIS and CIS */
-		switch (bt_iso_flags_pb(flags)) {
-		case BT_ISO_SINGLE:
-			pdu->ll_id = PDU_BIS_LLID_COMPLETE_END;
-			break;
-		default:
-			ll_iso_tx_mem_release(tx);
-			return -EINVAL;
-		}
-
-		pdu->len = slen;
-		memcpy(pdu->payload, buf->data, slen);
-
-		struct lll_adv_iso *lll_iso;
+		// FIXME: convey group start, sequence number
+		//sdu_frag_tx.grp_ref_point = 0;
+		sdu_frag_tx.target_event = 0;
 
-		lll_iso = &adv_iso->lll;
+		struct ll_iso_datapath *dp;
+		isoal_source_handle_t source;
```

Comment:
nitpick, reverse christmas tree, just a personal preference for declarations.
```suggestion
		isoal_source_handle_t source;
		struct ll_iso_datapath *dp;
```